src/Native/Unix/System.Native/pal_networking.c

```diff
@@ -2413,7 +2446,7 @@ int32_t SystemNative_SendFile(intptr_t out_fd, intptr_t in_fd, int64_t offset, i
     *sent = 0;
     return SystemNative_ConvertErrorPlatformToPal(errno);
 
-#elif HAVE_SENDFILE_6
+#elif HAVE_SENDFILE_6 && !defined(_AIX) /* why is AIX getting this? */
```

Comment:
The fix should be to update the cmake configuration however is necessary, not to add this !defined(_AIX) here.